app/Helpers/BackupEncryption.php

```diff
@@ -20,20 +20,20 @@ final class BackupEncryption
     /**
      * @var int
      */
-    private const ENCRYPTION_DEFAULT = ZipFile::ENCRYPTION_METHOD_TRADITIONAL;
+     const ENCRYPTION_DEFAULT = ZipFile::ENCRYPTION_METHOD_TRADITIONAL;
```

Comment:
I have no idea - It would prevent ...

`php artisan key:generate`
`php artisan migrate --seed`

... from running and if i reverted after that it would be the first error thrown.

Was run on two fresh homestead v10 boxes.